### ISSUE TITLE
codeceptjs init -- ENOENT: no such file or directory, mkdir

### ISSUE THREAD

**Issue by Zougi** [Codeception/CodeceptJS#259]
got this error on windows when specifying a folder for screenshots beneth the test folder that was not created yet

```
$ codeceptjs init

  Welcome to CodeceptJS initialization tool
  It will prepare and configure a test environment for you

Installing to C:\Users\zougi\Documents\GitHub\myproject
? Where are your tests located? ./test/e2e
? What helpers do you want to use? WebDriverIO, Nightmare
? Where should logs, screenshots, and reports to be stored? ./test/e2e/errorShots
? Would you like to extend I object with custom steps? No
? Do you want to choose localization for tests? English (no localization)
Configure helpers...
? [WebDriverIO] Base url of site to be tested http://localhost:8000
? [WebDriverIO] Browser in which testing will be performed chrome
? [Nightmare] Base url of site to be tested http://localhost:8000
Config created at C:\Users\zougi\Documents\GitHub\myproject\codecept.json
readline.js:930
            throw err;
            ^

Error: ENOENT: no such file or directory, mkdir 'C:\Users\zougi\Documents\GitHub\myproject\test\e2e\errorShots'
```
fyi `./test/e2e` folder was not created
CodeceptJS v0.4.7
NodeJS v4.5.0

**Comment by Zougi** [comment]
I believe it would have worked if I had set something like `./test/e2e/*.js` instead of `./test/e2e` .
Also I believe if I had set `./test/e2e/**/*.js` it may have created folder `e2e/**` because when I set value tests to `./test/e2e/**/*.js` in `codecept.json`,  I get error:
```
Error: ENOENT: no such file or directory, mkdir 'C:\Users\i301937\Documents\GitHub\wingy\test\e2e\**'
```

Can codeceptjs execute tests in folder benehth ``./test/e2e` !? like for example `./test/e2e/BAT`
Eitherway I believe there is something to do here.

**Comment by DavertMik** [comment]
Thanks for #260 and also please review my fix at #270 
At least it reproduces and fixes your issue
